lib/errors.d.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * URL in SitemapItem does not exists
+ */
+export declare class NoURLError extends Error {
+    constructor(message?: any);
+}
+/**
+ * Protocol in URL does not exists
+ */
+export declare class NoURLProtocolError extends Error {
+    constructor(message?: any);
+}
+/**
+ * changefreq property in sitemap is invalid
+ */
+export declare class ChangeFreqInvalidError extends Error {
+    constructor(message?: any);
+}
+/**
+ * priority property in sitemap is invalid
+ */
+export declare class PriorityInvalidError extends Error {
+    constructor(message?: any);
+}
+/**
+ * SitemapIndex target Folder does not exists
+ */
+export declare class UndefinedTargetFolder extends Error {
+    constructor(message?: any);
+}
+export declare class InvalidVideoFormat extends Error {
+    constructor(message?: any);
+}
+export declare class InvalidVideoDuration extends Error {
+    constructor(message?: any);
+}
+export declare class InvalidVideoDescription extends Error {
+    constructor(message?: any);
+}
+export declare class InvalidAttrValue extends Error {
+    constructor(key: any, val: any, validator: any);
```

Comment:
key is a string, val is any, validator is a regex. I'm not sure how to specify type as regex in typescript

Reply:
```suggestion
    constructor(key: string, val: any, validator: RegExp);
```